doc/stdenv/stdenv.chapter.md

```diff
@@ -425,6 +425,16 @@ A script to be run by `maintainers/scripts/update.nix` when the package is match
   };
   ```
 
+::: {.tip}
+A common pattern is to use the [`nix-update-script`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/common-updater/nix-update.nix) attribute provided in Nixpkgs, which runs [`nix-update`](https://github.com/Mic92/nix-update):
+
+```nix
+passhtru.updateScript = nix-update-script { };
+```
+
+For simple packages, this is often enough, and will ensure that the package is updated automatically by [`nixpkgs-update`](https://ryantm.github.io/nixpkgs-update) when a new version is released.
```

Comment:
Do you have to set `passthru.updateScript` in order for it to be run by `nixpkgs-update`? Is running `nixpkgs-update` a manual process or does a bot take care of it automatically?

Reply:
`passthru.updateScript` is not necessary if the project is listed on Repology.

It can be run manually but bot runs it periodically as well.

Reply:
> `passthru.updateScript` is not necessary if the project is listed on Repology.

Can we add that to the docs?

Reply:
That sounds great, I'll add a line tomorrow. I poked around in the nixpkgs-update documentation but I wasn't able to find this there, perhaps there's a documentation hole there too.